### PR TITLE
chore: add env support to CTR fetch and cache

### PR DESCRIPTION
**Pull request by Mrsirdev**
This pull request introduces the functionality to store and retrieve data from the CTR and its cache, based on an environment variable that distinguishes between production and staging environments.

### CODE REVIEW

File: cronjobs/scripts/price_entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (devops-evmos) [inline cronjobs/scripts/price_entrypoint.sh:1]: entrypoint was unified., we do not need this file anymore.

Comment (Mrsirdev) [reply cronjobs/scripts/price_entrypoint.sh]: Removed thanks!

File: cronjobs/github.py [hunk 2]
```diff
@@ -8,8 +8,9 @@
 from redis_functions import getChains, getTokens, setChains, setTokens
 
 
-CHAINS_DIRECTORY_URL = "https://api.github.com/repos/evmos/chain-token-registry/git/trees/main?recursive=1"
+ENVIRONMENT = os.getenv("ENVIRONMENT")
 
+CHAINS_DIRECTORY_URL = "https://api.github.com/repos/evmos/chain-token-registry/git/trees/production?recursive=1" if ENVIRONMENT == "production" else "https://api.github.com/repos/evmos/chain-token-registry/git/trees/main?recursive=1"
```
Comment (devops-evmos) [inline cronjobs/github.py:13]:
```suggestion
CHAINS_DIRECTORY_URL = f"https://api.github.com/repos/evmos/chain-token-registry/git/trees/{ENVIRONMENT if ENVIRONMENT == 'production' else 'main'}?recursive=1"
```

File: cronjobs/redis_functions.py [hunk 3]
```diff
@@ -20,8 +22,8 @@
     r = redis.Redis(decode_responses=True)
 
 
-erc20TokensDirectoryKey = "git-erc20-tokens-directory"
-networkConfig = "git-network-config-directory"
+erc20TokensDirectoryKey = "prod-git-erc20-tokens-directory" if ENVIRONMENT == "production" else "git-erc20-tokens-directory"
+networkConfig = "prod-git-network-config-directory" if ENVIRONMENT == "production" else "git-network-config-directory"
```
Comment (devops-evmos) [inline cronjobs/redis_functions.py:26]:
```suggestion
prod_prefix = "prod-" if ENVIRONMENT == "production" else ""

erc20TokensDirectoryKey = f"{prod_prefix}git-erc20-tokens-directory"
networkConfig = f"{prod_prefix}git-network-config-directory"
```

File: internal/v1/db/erc20.go [hunk 4]
```diff
@@ -34,25 +41,25 @@ func RedisGetERC20Balance(contract string, address string) (string, error) {
 }
 
 func RedisSetERC20TokensDirectory(result string) {
-	err := rdb.Set(ctxRedis, erc20TokensDirectoryKey, result, time.Duration(oneDayExpiration*int(time.Second))).Err()
+	err := rdb.Set(ctxRedis, getErc20TokensDirectoryKey(), result, time.Duration(oneDayExpiration*int(time.Second))).Err()
```
Comment (facs95) [inline internal/v1/db/erc20.go:44]: Just for the code eto be more readable, can you create a variable of the key first ?

Comment (facs95) [reply internal/v1/db/erc20.go:44]: Another optionis to have this instantiated in an init function so you dont have to call it on every function

File: internal/v1/db/network.go [hunk 5]
```diff
@@ -4,38 +4,60 @@
...
 	"time"
 )
 
-// func buildNetwokrKey(url string) string {
-// 	var sb strings.Builder
-// 	sb.WriteString("githubcache")
-// 	sb.WriteString(url)
-// 	return sb.String()
-// }
+//	func buildNetwokrKey(url string) string {
```
Comment (sandoche) [inline internal/v1/db/network.go:12]: Should we delete those commented code?

Comment (Mrsirdev) [reply internal/v1/db/network.go]: Done!

**Pull request merged**